packages/ui/src/domains/LandingPageBuilder/LandingPageBuilder.tsx

```
@@ -0,0 +1,2554 @@
+		}
+	};
+
+	const addComponent = (
+		containerIndex: number,
+		type: ComponentType,
+		subContainerIndex?: number,
+	) => {
+		const newContainers = [...containers];
+		const targetContainer =
+			subContainerIndex !== undefined
+				? (newContainers[containerIndex]?.components[subContainerIndex]!
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)